content/blog/2025-04-28-datafusion-47.0.0.md

```diff
@@ -0,0 +1,240 @@
+---
+layout: post
+title: Apache DataFusion 45.0.0 Released
+date: 2025-02-20
+author: pmc
+categories: [release]
+---
+
```

Comment:
FYI @Omega359  said he was working on some posts -- I think the plan is that he will make separate PRs for the 47 . 48 and 49 releases and we'll retroactively publish posts about those releases


Once he has that skeleton I'll port any additional content that might be relevant from this PR and close it